### PR TITLE
use ansible DEFAULT_LOCAL_TMP for files stored on the ansible controller

### LINKED ISSUE #354: [Bug]: role is not multi user aware due to hard coded file in /tmp
### Please try to fill out as much of the information below as you can. Thank you!

- [X] Yes, I've searched similar issues on GitHub and didn't find any.

### Which version contains the bug?

_No response_

### Describe the bug

 * role: netways.elasticstack.elasticsearch
 * task: Fetch ca certificate from ca host to Ansible controller
 * error:
`FAILED! => {"changed": false, "msg": "Failed to fetch the file: [Errno 13] Permission denied: b'/tmp/ca.crt'"`

### How to recreate the bug?

run the role under 2 different users on the same ansible controller

### PR DESCRIPTION
**Pull request by zerwes**
https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-local-tmp

closes #354

### CODE REVIEW

File: roles/elasticsearch/tasks/elasticsearch-security.yml [hunk 1]
```diff
@@ -185,20 +185,20 @@
   block:
     - name: Check the existance of cert on Ansible controler
       ansible.builtin.stat:
-        path: /tmp/{{ ansible_hostname }}.p12
+        path:  "{{ lookup('config', 'DEFAULT_LOCAL_TMP') | dirname }}/{{ ansible_hostname }}.p12"
```
Comment (widhalmt) [inline roles/elasticsearch/tasks/elasticsearch-security.yml:188]:
```suggestion
        path: "{{ lookup('config', 'DEFAULT_LOCAL_TMP') | dirname }}/{{ ansible_hostname }}.p12"
```

I just foud a little bit of lint. ;-)

File: roles/elasticsearch/tasks/elasticsearch-security.yml [hunk 2]
```diff
@@ -185,20 +185,20 @@
...
       register: elasticsearch_check_temporary_cert
 
     - name: Move temporary cert on Ansible controler
       ansible.builtin.copy:
-        src: /tmp/{{ ansible_hostname }}.p12
-        dest: "/tmp/{{ ansible_hostname }}.p12_{{ ansible_date_time.iso8601_micro }}"
+        src: "{{ lookup('config', 'DEFAULT_LOCAL_TMP') | dirname }}/{{ ansible_hostname }}.p12"
+        dest:  "{{ lookup('config', 'DEFAULT_LOCAL_TMP') | dirname }}/{{ ansible_hostname }}.p12_{{ ansible_date_time.iso8601_micro }}"
```
Comment (widhalmt) [inline roles/elasticsearch/tasks/elasticsearch-security.yml:194]:
```suggestion
        dest: "{{ lookup('config', 'DEFAULT_LOCAL_TMP') | dirname }}/{{ ansible_hostname }}.p12_{{ ansible_date_time.iso8601_micro }}"
```
Same with extra spaces here.

File: roles/elasticsearch/tasks/elasticsearch-security.yml [hunk 3]
```diff
@@ -185,20 +185,20 @@
...
         mode: preserve
       when: elasticsearch_check_temporary_cert.stat.exists
       register: elasticsearch_move_cert_file
 
     - name: Remove temporary cert on Ansible controler
       ansible.builtin.file:
-        path: /tmp/{{ ansible_hostname }}.p12
+        path:  "{{ lookup('config', 'DEFAULT_LOCAL_TMP') | dirname }}/{{ ansible_hostname }}.p12"
```
Comment (widhalmt) [inline roles/elasticsearch/tasks/elasticsearch-security.yml:201]:
```suggestion
        path: "{{ lookup('config', 'DEFAULT_LOCAL_TMP') | dirname }}/{{ ansible_hostname }}.p12"
```
I think, I see a pattern. :-)

**Review by widhalmt — CHANGES_REQUESTED** [review]
Oh, very good find! Thank you for the contribution.

I just found some tiny bits of lint. Could you please fix them and then I'll merge.

File: roles/elasticsearch/tasks/elasticsearch-security.yml (hunk 3 above)
Comment (zerwes) [reply roles/elasticsearch/tasks/elasticsearch-security.yml:201]: yes, somehow my search/replace vim regex seem to had a hiccup ... or it was rather pecak and not regex? Anyhow, thank you for you attention and suggestion.

**Comment by zerwes** [comment]
> Oh, very good find! Thank you for the contribution.
> 
> I just found some tiny bits of lint. Could you please fix them and then I'll merge.

Thank you @widhalmt for the hint and suggestions and sorry for the error.
Now a `git grep ':  "{{'` in the repo is silent.

**Pull request merged**